core/src/test/scala/com/microsoft/azure/synapse/ml/nbtest/SynapseTests.scala

```diff
@@ -155,6 +154,7 @@ class SynapseTests extends TestBase {
       failures.foreach(failure =>
         println(failure.failed.get.getMessage))
     }
+    FileUtils.deleteDirectory(notebooksDir)
```

Comment:
Donmt need this line as its customary to leave generated code in target directory for debugging purposes